src/Components/BranchesControl.jsx

```diff
@@ -49,6 +52,11 @@ export default function Branches() {
           versionPathsTemp.push(versionPath)
         })
         setVersionPaths(versionPathsTemp)
+        if (branchesData.length > 1) {
```

Comment:
this does not change the previous logic of the branch control, just adds a store value used in control group.